docs/example-react-hooks-useReducer.md

```diff
@@ -0,0 +1,78 @@
+Basic example showing how to test useReducer hook. 
+
+
+The component we are testing which changes some text depending on `stateprop1` value.
+
+```jsx
+import React, { useReducer } from 'react'
+
+const initialState = {
+    isConfirmed: false,
+}
+
+function reducer(state = initialState, action) {
+  switch(action.type) {
+    case "SUCCESS":
+      return {
+        ...state,
+        isConfirmed: true,
+      }
+    case "FAILURE":
```

Comment:
If the "FAILURE" state isn't being used, I'd remove it from the demo, so code is as focused on the topic as it can be :)